site/public/templates/grading/EditComponent.twig

```diff
@@ -97,14 +97,14 @@
             <div class="col-label-fixed">How should points be assigned?</div>
             <div class="col">
                 <span class="radio-input">
-                    <input type="radio" class="count-up-selector count-type-selector"
+                    <input type="radio" class="count-up-selector count-type-selector key_to_click" tabindex="0"
                            name="count-type-{{ component.id }}" {{ component.default == 0 ? 'checked' : '' }}
                            onclick="onClickCountUp(this)" id="grade_by_count_up"> <label for="grade_by_count_up">Grade by Count Up (from zero)</label>
                 </span>
             </div>
             <div class="col">
                 <span class="radio-input">
-                    <input type="radio" class="count-down-selector count-type-selector"
+                    <input type="radio" class="count-down-selector count-type-selector key_to_click" tabindex="0"
                            name="count-type-{{ component.id }}" {{ component.default != 0 ? 'checked' : '' }}
                            onclick="onClickCountDown(this)" id="grade_by_count_down"> <label for="grade_by_count_down">Grade by Count Down (from "Points")</label>
```

Comment:
Not *entirely* sure whether this is necessary; it seems like when radio buttons are concerned, one group counts as one "entry" in the tab ordering and switching between items is accomplished with the arrow keys.